apps/ui/lib/components/Event/index.tsx

```diff
@@ -14,35 +14,78 @@ import Message from './Message';
 export { parseMessage };
 
 const EventWithActor: React.FunctionComponent<any> = (props) => {
-	const { card, user, onCardVisible, targetCard } = props;
+	const { card, user, onCardVisible, targetCard, isDirectMention } = props;
+	const mentionsUserArray = card.data.payload?.mentionsUser || [];
+	const directMessageMarkersArray = card.markers || [];
+	const isDirectMessagePing =
+		directMessageMarkersArray[0]?.split('+')[0] === user.slug || false;
```

Comment:
Are there cases in which `card.markers` does have a `+`? If so, then that means multiple entities are assigned in the `markers` and we probably can't assume its always the first in the list (like you're doing with `[0]`.